mod_test.go

```diff
@@ -0,0 +1,68 @@
+package dyspatch_test
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/antihax/optional"
+	"github.com/getdyspatch/dyspatch-golang"
+	"golang.org/x/net/context"
+)
+
+const version = "application/vnd.dyspatch.2019.10+json"
+
+func bodyToString(body io.ReadCloser) string {
+	buf := new(bytes.Buffer)
+	buf.ReadFrom(body)
+	return buf.String()
+}
+
+func TestMain(t *testing.T) {
```

Comment:
are these generated tests, or manual?

Reply:
These are manual, ya I definitely could clean some of them up copied some code Dylan had into here.

Reply:
up to you, would be a nice improvement is all.

Reply:
I'll add a task into the backlog for API client tests. Just to push along the work at the moment.